src/main/java/com/bb3/bodybuddybe/users/UsersRoleEnum.java

```diff
@@ -0,0 +1,28 @@
+package com.bb3.bodybuddybe.users;
+
+public enum UsersRoleEnum {
+    USER(Authority.USER),
+    CHATOWNER(Authority.CHATOWNER),
+    POSTOWNER(Authority.POSTOWNER),
```

Comment:
상수에 언더바를 넣으면 가독성이 더 좋아질 것 같아요 !
```suggestion
    CHAT_OWNER(Authority.CHAT_OWNER),
    POST_OWNER(Authority.POST_OWNER),
```

Reply:
네!! 해당부분으로 변경하는 것이 가독성이 더 좋아보입니다ㅎㅎ 반영해봐야겠어요!